tools/mtouch/Makefile

```diff
@@ -172,6 +172,8 @@ SIMLAUNCHER_FRAMEWORKS =  \
 						\
 	-weak_framework CoreLocationUI \
 	-weak_framework OSLog \
+						\
+	-weak_framework SharedWithYouCore \
```

Comment:
When do we add this here?

Reply:
Read the apple docs here: https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WeakLinking.html

and this answer in stack overflow: https://stackoverflow.com/questions/16922013/what-does-it-mean-to-weak-link-a-framework

Reply:
Cool makes sense thanks!